### ISSUE TITLE
🔦 Feature: Support enabling invite-only signup via `NC_INVITE_ONLY_SIGNUP` env var again

### ISSUE THREAD

**Issue by salim-b** [nocodb/nocodb#7814]
### Please confirm if feature request does NOT exist already ?

- [X] I confirm there is no existing issue for this

### Describe the usecase for the feature

Before NocoDB v0.99.0, it was possible to disable public signup (i.e. enable invite-only signup) by setting the runtime environment variable `NC_INVITE_ONLY_SIGNUP=true`. This was very handy as no manual post-deployment UI action was necessary to secure a new NocoDB installation.

### Suggested Solution

Restore enabling NocoDB's the invite-only signup option via setting `NC_INVITE_ONLY_SIGNUP=true` again.

**Comment by pranavxc** [comment]
This is still possible from UI for super admin user.


https://docs.nocodb.com/0.109.7/setup-and-usages/account-settings/

**Comment by salim-b** [comment]
> This is still possible from UI for super admin user.

Yes, I know. But this requires configuration via the NocoDB UI. I'd love to be able to set this option via the `NC_INVITE_ONLY_SIGNUP` runtime variable. 🙂

**Comment by erem2k** [comment]
+, that would be really helpful as one of the defaults for automated deployment scenarios
